lib/http_requests.py

```diff
@@ -52,128 +48,74 @@ def get_all_ocp_build_data_branches():
 
 
 def get_branch_details_for_ocp_build_data(branch_name):
-
     branch_group_yml_url = get_group_yml_file_url(branch_name)
     return branch_group_yml_url
 
 
 def get_group_yml_file_url(branch_name: str) -> dict:
-
     """
     This method takes a branch name of ocp_build_data as a parameter and returns advisories details for the same.
     :param branch_name: Branch name of ocp_build_data
     :return: adivsories details
     """
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
     hit_url = app_constants.GITHUB_GROUP_YML_CONTENTS_URL.format(branch_name)
-    hit_request = requests.get(hit_url, headers={"Authorization": "token " + access_token})
+    hit_request = requests.get(hit_url)
     return hit_request.json()["download_url"]
 
 
 def get_github_rate_limit_status():
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
-    hit_request = requests.get(os.environ["GITHUB_RATELIMIT_ENDPOINT"], headers={"Authorization": "token " + access_token})
+    hit_request = requests.get(os.environ["GITHUB_RATELIMIT_ENDPOINT"])
     hit_response = hit_request.json()
     hit_response = hit_response["rate"]
     hit_response["reset_secs"] = hit_response["reset"] - time.time()
-    hit_response["reset_mins"] = int((hit_response["reset"] - time.time())/60)
+    hit_response["reset_mins"] = int((hit_response["reset"] - time.time()) / 60)
     return hit_response
 
 
 def get_advisory_ids_from_sha(sha):
     group_yml_url = os.environ["GITHUB_RAW_CONTENT_URL"].format(sha)
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
-    hit_request = requests.get(group_yml_url,
-                               headers={"Authorization": "token " + access_token})
+    hit_request = requests.get(group_yml_url)
     hit_response = yaml.load(hit_request.text)
     if "advisories" in hit_response:
         return hit_response["advisories"]
     return {}
 
 
+def get_advisories(branch_name):
+    yml_file = f"https://raw.githubusercontent.com/openshift/ocp-build-data/{branch_name}/releases.yml"
+    response = requests.get(yml_file)
+    yml_data = yaml.load(response.text, Loader=yaml.Loader)['releases']
+
+    advisory_data = []
+    for version in yml_data:
+        try:
+            advisories = yml_data[version]['assembly']['group']['advisories']
+            brew_event = yml_data[version]['assembly']['basis']['brew_event']
+        except:
+            continue
+        # advisory_data += list(advisories)
+        if -1 in advisories or 1 in advisories:
+            continue
```

Comment:
`-1` or `1` will not be keys in the advisories, they'll be values, so...
```suggestion
        if -1 in advisories or 1 in advisories.values():
            continue
```